<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Publications | Neurodevelopment and Behaviour Lab </title> <meta name="author" content="Neurodevelopment and Behaviour "> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="tigs, soniasen, sonia-sen, neurodevelopment, prashali-bansal, prahsali, mosquito, omics,"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?b05f9a0b7405d7c8c89c7465593dea81"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?dadeb9c5d1fd12bc8d37475657446863"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?1fefb6021bf36010f25ea1cef24af84e"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?53a094b51ed1d1e025731eb00d240058" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://soniasen.github.io/publications/"> <script src="/assets/js/theme.js?f2531f05c6f8e1622518f4f5a1e385b1"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?46af317e693b09921dcb92261d123fbc" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Neurodevelopment and Behaviour Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">Projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/news/">News </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">People </a> </li> <li class="nav-item "> <a class="nav-link" href="/join_us/">Join Us </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description"></p> </header> <article> <script src="/assets/js/bibsearch.js?8470e568bd8f7fe1a4046936bc17c40d" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div id="BANSAL2025103070" class="col-sm-10"> <div class="title">Sexual dimorphism in the behaviour and sensory systems of mosquitoes</div> <div class="author"> Prashali Bansal and Sonia Q. Sen </div> <div class="periodical"> <em>Current Opinion in Neurobiology</em>, Aug 2025 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/https://doi.org/10.1016/j.conb.2025.103070" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Bansal_2025" class="col-sm-10"> <div class="title">Two neuropeptides that promote blood-feeding in Anopheles stephensi mosquitoes</div> <div class="author"> Prashali Bansal, Roshni Pillai, Pooja D Babu, and Sonia Q Sen </div> <div class="periodical"> <em>eLife</em>, Oct 2025 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.7554/elife.108625.1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Bhattacharya2025.11.25.690435" class="col-sm-10"> <div class="title">Chromatin priming and Hunchback recruitment integrate spatial and temporal cues in Drosophila neuroblasts</div> <div class="author"> Ayanthi Bhattacharya, Hemalatha Rao, and Sonia Q Sen </div> <div class="periodical"> <em>bioRxiv</em>, Nov 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2025.11.25.690435" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Neural stem cells generate diverse cell types by integrating spatial and temporal cues to activate neuron-specific terminal selector (TS) genes. In Drosophila neuroblasts (NBs), spatial patterning sets lineage identity, while a temporal transcription factor (TTF) cascade sets birth order. Two proposed mechanisms could integrate these inputs. In direct regulation, spatial transcription factors (STFs) and TTFs co-occupy and activate TS enhancers within NBs. In epigenetic regulation, STFs first prime NB-specific chromatin, creating ’sites of integration’ (SoIs) that later recruit TTFs. We test this in two identified NBs — NB5-6 and NB7-4 — and their candidate STFs, Gooseberry (Gsb) and Engrailed (En), together with the first TTF, Hunchback (Hb). In NB5-6, Gsb is expressed transiently, suggesting a chromatin-based memory of its activity. In NB7-4, En expression persists throughout development so integration could either be epigenetic or direct. We used chromatin engagement by the STFs as the discriminator between these models. If integration is epigenetic, the STF must engage less-accessible chromatin to establish NB-specific SoIs; if regulation is direct, the STF need not. We find that En binds only to pre-accessible loci in NB7-4 and En+Hb co-binding marks the most accessible enhancers. This suggests that NB7-4 likely relies on an unknown priming factor to establish SoIs, with direct En-Hb co-binding mediating enhancer activation. In NB5-6, Gsb binds both open and less-accessible chromatin and Gsb+Hb co-binding marks the most accessible enhancers. When ectopically expressed, Gsb remodels chromatin globally in the non-cognate NB7-4, and at endogenous NB7-4-SoIs, it specifically reduces accessibility as well as Hb binding. This suggests that in NB5-6 Gsb likely acts together with other NB5-6-specific factors to recognize less-accessible chromatin and to promote Hb recruitment while restricting Hb occupancy to appropriate enhancers. Together these findings support a unified two-step model: NB-specific combinations of TFs — each NB’s "STF code" — first prime chromatin and then recruit and restrict Hb to ensure lineage-specific enhancer activation.Competing Interest StatementThe authors have declared no competing interest.Ramalingaswami FellowshipUGC-CSIRTIGS</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div id="Sens2024" class="col-sm-10"> <div class="title">Transitions in development - an interview with Sonia Sen</div> <div class="author"> Sonia Sen </div> <div class="periodical"> <em>Development (Cambridge)</em>, Jul 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1242/DEV.203137/359253" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div id="SEN202354" class="col-sm-10"> <div class="title">Generating neural diversity through spatial and temporal patterning</div> <div class="author"> Sonia Q. Sen </div> <div class="periodical"> <em>Seminars in Cell &amp; Developmental Biology</em>, Jul 2023 </div> <div class="periodical"> Special Issue: Temporal patterning in the CNS </div> <div class="links"> <a href="https://doi.org/https://doi.org/10.1016/j.semcdb.2022.06.002" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div id="10.7554/eLife.44036" class="col-sm-10"> <div class="title">Neuroblast-specific open chromatin allows the temporal transcription factor, Hunchback, to bind neuroblast-specific loci</div> <div class="author"> Sonia Q Sen, Sachin Chanchani, Tony D Southall, and Chris Q Doe </div> <div class="periodical"> <em>eLife</em>, Jan 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/eLife.44036" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Spatial and temporal cues are required to specify neuronal diversity, but how these cues are integrated in neural progenitors remains unknown. \textitDrosophila progenitors (neuroblasts) are a good model: they are individually identifiable with relevant spatial and temporal transcription factors known. Here we test whether spatial/temporal factors act independently or sequentially in neuroblasts. We used Targeted DamID to identify genomic binding sites of the Hunchback temporal factor in two neuroblasts (NB5-6 and NB7-4) that make different progeny. Hunchback targets were different in each neuroblast, ruling out the independent specification model. Moreover, each neuroblast had distinct open chromatin domains, which correlated with differential Hb-bound loci in each neuroblast. Importantly, the Gsb/Pax3 spatial factor, expressed in NB5-6 but not NB7-4, had genomic binding sites correlated with open chromatin in NB5-6, but not NB7-4. Our data support a model in which early-acting spatial factors like Gsb establish neuroblast-specific open chromatin domains, leading to neuroblast-specific temporal factor binding and the production of different neurons in each neuroblast lineage.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Sen2019" class="col-sm-10"> <div class="title">Neurotransmitter identity: A question of lineage</div> <div class="author"> Sonia Sen </div> <div class="periodical"> <em>eLife</em>, May 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/ELIFE.47162" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>In the ventral nerve cord of fruit flies, neurons from the same hemilineage use the same neurotransmitter.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Reichert2019" class="col-sm-10"> <div class="title">Heinrich Reichert (1949-2019)</div> <div class="author"> Heinrich Reichert, Sonia Sen, and K Vijayraghavan </div> <div class="periodical"> <em>Development</em>, Aug 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1242/DEV.183517" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Heinrich Reichert, Professor Emeritus at the University of Basel, Switzerland, passed away on the 13th of June 2019 after a prolonged illness. Heinrich described himself as ‘a hedonist when it came to science’ because he said it gave him great pleasure. It was this quality that made working with Heinrich thrilling and deeply fulfilling. Heinrich’s long and versatile career spanned the breadth of neuroscience – from development, to evolution and behaviour. In his passing we have lost not just an astute scientist, but also an impassioned educator and an adventurer of science.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div id="Arboleda2018" class="col-sm-10"> <div class="title">An Emerging System to Study Photosymbiosis, Brain Regeneration, Chronobiology, and Behavior: The Marine Acoel Symsagittifera roscoffensis</div> <div class="author"> Enrique Arboleda, Volker Hartenstein, Pedro Martinez, Heinrich Reichert, Sonia Sen, Simon Sprecher, and Xavier Bailly </div> <div class="periodical"> <em>BioEssays</em>, Oct 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1002/BIES.201800107;SUBPAGE:STRING:ABSTRACT;WEBSITE:WEBSITE:PERICLES;JOURNAL:JOURNAL:15211878;WGROUP:STRING:PUBLICATION" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The acoel worm Symsagittifera roscoffensis, an early offshoot of the Bilateria and the only well-studied marine acoel that lives in a photosymbiotic relationship, exhibits a centralized nervous system, brain regeneration, and a wide repertoire of complex behaviors such as circatidal rhythmicity, photo/geotaxis, and social interactions. While this animal can be collected by the thousands and is studied historically, significant progress is made over the last decade to develop it as an emerging marine model. The authors here present the feasibility of culturing it in the laboratory and describe the progress made on different areas, including genomic and tissue architectures, highlighting the associated challenges. In light of these developments, and on the ability to access abundant synchronized embryos, the authors put forward S. roscoffensis as a marine system to revisit questions in the areas of photosymbiosis, regeneration, chronobiology, and the study of complex behaviors from a molecular and evolutionary perspective.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Sen2014" class="col-sm-10"> <div class="title">Orthodenticle is required for the development of olfactory projection neurons and local interneurons in Drosophila</div> <div class="author"> Sonia Sen, Silvia Biagini, Heinrich Reichert, and K. VijayRaghavan </div> <div class="periodical"> <em>Biology Open</em>, Aug 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1242/BIO.20148524" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The accurate wiring of nervous systems involves precise control over cellular processes like cell division, cell fate specification, and targeting of neurons. The nervous system of Drosophila melanogaster is an excellent model to understand these processes. Drosophila neurons are generated by stem cell like precursors called neuroblasts that are formed and specified in a highly stereotypical manner along the neuroectoderm. This stereotypy has been attributed, in part, to the expression and function of transcription factors that act as intrinsic cell fate determinants in the neuroblasts and their progeny during embryogenesis. Here we focus on the lateral neuroblast lineage, ALl1, of the antennal lobe and show that the transcription factorencoding cephalic gap gene orthodenticle is required in this lineage during postembryonic brain development. We use immunolabelling to demonstrate that Otd is expressed in the neuroblast of this lineage during postembryonic larval stages. Subsequently, we use MARCM clonal mutational methods to show that the majority of the postembryonic neuronal progeny in the ALl1 lineage undergoes apoptosis in the absence of orthodenticle. Moreover, we demonstrate that the neurons that survive in the orthodenticle loss-of-function condition display severe targeting defects in both the proximal (dendritic) and distal (axonal) neurites. These findings indicate that the cephalic gap gene orthodenticle acts as an important intrinsic determinant in the ALl1 neuroblast lineage and, hence, could be a member of a putative combinatorial code involved in specifying the fate and identity of cells in this lineage.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Sen2015" class="col-sm-10"> <div class="title">Genetic transformation of structural and functional circuitry rewires the Drosophila brain</div> <div class="author"> Sonia Sen, Deshou Cao, Ramveer Choudhary, Silvia Biagini, Jing W. Wang, Heinrich Reichert, and K. VijayRaghavan </div> <div class="periodical"> <em>eLife</em>, Aug 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/ELIFE.04407" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Acquisition of distinct neuronal identities during development is critical for the assembly of diverse functional neural circuits in the brain. In both vertebrates and invertebrates, intrinsic determinants are thought to act in neural progenitors to specify their identity and the identity of their neuronal progeny. However, the extent to which individual factors can contribute to this is poorly understood. We investigate the role of orthodenticle in the specification of an identified neuroblast (neuronal progenitor) lineage in the Drosophila brain. Loss of orthodenticle from this neuroblast affects molecular properties, neuroanatomical features, and functional inputs of progeny neurons, such that an entire central complex lineage transforms into a functional olfactory projection neuron lineage. This ability to change functional macrocircuitry of the brain through changes in gene expression in a single neuroblast reveals a surprising capacity for novel circuit formation in the brain and provides a paradigm for large-scale evolutionary modification of circuitry.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"><li> <div class="row"> <div id="Sen2013" class="col-sm-10"> <div class="title">Conserved roles of ems/Emx and otd/Otx genes in olfactory and visual system development in Drosophila and mouse</div> <div class="author"> Sonia Sen, Heinrich Reichert, and K. VijayRaghavan </div> <div class="periodical"> <em>Open Biology</em>, Aug 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1098/RSOB.120177;JOURNAL:JOURNAL:RSOB;PAGEGROUP:STRING:PUBLICATION" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The regional specialization of brain function has been well documented in the mouse and fruitfly. The expression of regulatory factors in specific regions of the brain during development suggests that they function to establish or maintain this specialization. Here, we focus on two such factors-the Drosophila cephalic gap genes empty spiracles (ems) and orthodenticle (otd), and their vertebrate homologues Emx1/2 and Otx1/2-and review novel insight into their multiple crucial roles in the formation of complex sensory systems. While the early requirement of these genes in specification of the neuroectoderm has been discussed previously, here we consider more recent studies that elucidate the later functions of these genes in sensory system formation in vertebrates and invertebrates. These new studies show that the ems and Emx genes in both flies and mice are essential for the development of the peripheral and central neurons of their respective olfactory systems. Moreover, they demonstrate that the otd and Otx genes in both flies and mice are essential for the development of the peripheral and central neurons of their respective visual systems. Based on these recent experimental findings, we discuss the possibility that the olfactory and visual systems of flies and mice share a common evolutionary origin, in that the conserved visual and olfactory circuit elements derive from conserved domains of otd/Otx and ems/Emx action in the urbilaterian ancestor. © 2013 The Authors.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2010</h2> <ol class="bibliography"><li> <div class="row"> <div id="Sen2010" class="col-sm-10"> <div class="title">Expression and function of the empty spiracles gene in olfactory sense organ development of Drosophila melanogaster</div> <div class="author"> Sonia Sen, Beate Hartmann, Heinrich Reichert, and Veronica Rodrigues </div> <div class="periodical"> <em>Development</em>, Nov 2010 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1242/DEV.052407" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>In Drosophila, the cephalic gap gene empty spiracles plays key roles in embryonic patterning of the peripheral and central nervous system. During postembryonic development, it is involved in the development of central olfactory circuitry in the antennal lobe of the adult. However, its possible role in the postembryonic development of peripheral olfactory sense organs has not been investigated. Here, we show that empty spiracles acts in a subset of precursors that generate the olfactory sense organs of the adult antenna. All empty spiracles-expressing precursor cells co-express the proneural gene amos and the early patterning gene lozenge. Moreover, the expression of empty spiracles in these precursor cells is dependent on both amos and lozenge. Functional analysis reveals two distinct roles of empty spiracles in the development of olfactory sense organs. Genetic interaction studies in a lozenge-sensitized background uncover a requirement of empty spiracles in the formation of trichoid and basiconic olfactory sensilla. MARCM-based clonal mutant analysis reveals an additional role during axonal targeting of olfactory sensory neurons to glomeruli within the antennal lobe. Our findings on empty spiracles action in olfactory sense organ development complement previous studies that demonstrate its requirement in olfactory interneurons and, taken together with studies on the murine homologs of empty spiracles, suggest that conserved molecular genetic programs might be responsible for the formation of both peripheral and central olfactory circuitry in insects and mammals. © 2010. Published by The Company of Biologists Ltd.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2009</h2> <ol class="bibliography"><li> <div class="row"> <div id="sen2009drosophila" class="col-sm-10"> <div class="title">Drosophila Olfactory Local Interneurons and Lateral Projection Neurons arise from a Single Lineage.</div> <div class="author"> Sonia Sen, Abhijit Das, Robert Lichtneckert, Kei Ito, Veronica Rodrigues, and Heinrich Reichert </div> <div class="periodical"> <em>In JOURNAL OF NEUROGENETICS</em>, Nov 2009 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> <h2 class="bibliography">2008</h2> <ol class="bibliography"><li> <div class="row"> <div id="Das2008" class="col-sm-10"> <div class="title">Drosophila olfactory local interneurons and projection neurons derive from a common neuroblast lineage specified by the empty spiracles gene</div> <div class="author"> Abhijit Das, Sonia Sen, Robert Lichtneckert, Ryuichi Okada, Kei Ito, Veronica Rodrigues, and Heinrich Reichert </div> <div class="periodical"> <em>Neural Development 2008 3:1</em>, Dec 2008 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1186/1749-8104-3-33" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Encoding of olfactory information in insects occurs in the antennal lobe where the olfactory receptor neurons interact with projection neurons and local interneurons in a complex sensory processing circuitry. While several studies have addressed the developmental mechanisms involved in specification and connectivity of olfactory receptor neurons and projection neurons in Drosophila, the local interneurons are far less well understood. In this study, we use genetic marking techniques combined with antibody labelling and neuroblast ablation to analyse lineage specific aspects of local interneuron development. We find that a large set of local interneurons labelled by the GAL4-LN1 (NP1227) and GAL4-LN2 (NP2426) lines arise from the lateral neuroblast, which has also been shown to generate uniglomerular projection neurons. Moreover, we find that a remarkable diversity of local interneuron cell types with different glomerular innervation patterns and neurotransmitter expression derives from this lineage. We analyse the birth order of these two distinct neuronal types by generating MARCM (mosaic analysis with a repressible cell marker) clones at different times during larval life. This analysis shows that local interneurons arise throughout the proliferative cycle of the lateral neuroblast beginning in the embryo, while uniglomerular projection neurons arise later during the second larval instar. The lateral neuroblast requires the function of the cephalic gap gene empty spiracles for the development of olfactory interneurons. In empty spiracles null mutant clones, most of the local interneurons and lateral projection neurons are lacking. These findings reveal similarities in the development of local interneurons and projection neurons in the olfactory system of Drosophila. We find that the lateral neuroblast of the deutocerebrum gives rise to a large and remarkably diverse set of local interneurons as well as to projection neurons in the antennal lobe. Moreover, we show that specific combinations of these two neuron types are produced in specific time windows in this neuroblast lineage. The development of both these cell types in this lineage requires the function of the empty spiracles gene.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="sticky-bottom mt-5" role="contentinfo"> <div class="container"> © Copyright 2025 Neurodevelopment and Behaviour . Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Website created by Ankit Jetwani and Prashali Bansal. Illustrations by Prashali Bansal. Last updated: December 11, 2025. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?c999e8c5281874b7534e3352f835d4c3" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?b977fe0c21b2118ed853308b1b923969"></script> <script src="/assets/js/no_defer.js?699fa7cbe3b29f831db7d5250ba3203a"></script> <script defer src="/assets/js/common.js?d3a25b46bbd2e0a751a27b173abc6e5f"></script> <script defer src="/assets/js/copy_code.js?fff63901a03063094790ffbfd4bc0cb4" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?25eff8ff4144a010e4ad7b31403102cf"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?04761245f225e3ad9e5e3f875d4e1074"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?b78cb42895d74e4fd6de6f633edcf181" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> </body> </html>